docs/lib/analytics/fragments/android/getting-started/30_initAnalytics.md

```diff
@@ -0,0 +1,17 @@
+Add the following code to the onCreate() method of `MainActivity.java`
+
+```java
+try {
+        AmazonCognitoAuthPlugin authPlugin = new AmazonCognitoAuthPlugin((Application) context)
+        AmazonPinpointAnalyticsPlugin analyticsPlugin = new AmazonPinpointAnalyticsPlugin((Application) context);
+        Amplify.addPlugin(authPlugin)
+        Amplify.addPlugin(analyticsPlugin);
+        Amplify.configure(context);
+        Log.i("GetStarted", "Amplify initialized with auth and analytics plugins");
+    } catch (Exception e) {
+        Log.e("GetStarted", "Error initializing", e);
+    }
```

Comment:
`context` isn't necessary. Have them add these lines to the `Application` class instead of `MainActivity.java`.

Rewrite:

Add the following code to your onCreate() method in your application class:

```java
Amplify.addPlugin(new AWSCognitoAuthPlugin());
Amplify.addPlugin(new AWSPinpointAnalyticsPlugin());
```

Your class will look like this:

```java
public class MyAmplifyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        try {
            // Add these lines to add the AWSCognitoAuthPlugin and AWSPinpointAnalyticsPlugin plugins
            Amplify.addPlugin(new AWSCognitoAuthPlugin());
            Amplify.addPlugin(new AWSPinpointAnalyticsPlugin());
            Amplify.configure(getApplicationContext());

            Log.i("MyAmplifyApplication", "Initialized Amplify");
        } catch (AmplifyException error) {
            Log.e("MyAmplifyApplication", "Could not initialize Amplify", error);
        }
    }
}
```

Reply:
Changed